packages/grid/_modules_/grid/hooks/features/pagination/gridPaginationReducer.ts

```diff
@@ -88,7 +88,7 @@ export const setGridRowCountStateUpdate = (state, payload): GridPaginationState
       ...state,
       pageCount: newPageCount,
       rowCount: totalRowCount,
-      page: state.page > newPageCount ? newPageCount - 1 : state.page,
+      page: Math.min(state.page, newPageCount - 1),
```

Comment:
How is that different 🤔

Reply:
min is equivalent to:

```suggestion
      page: state.page > newPageCount -1 ? newPageCount - 1 : state.page,,
```

Reply:
I have used `m`in because it makes it impossible for this error to happen, and since it went through two reviews, I thought that we might as well enforce it. No real preference.